practical/matlab/idr0021_steps.m

```diff
@@ -140,6 +150,8 @@
         roi.addShape(shape);
         area = polyarea(x_coordinates, y_coordinates);
         max_area = max(max_area, area);
+        max_per = max(max_per, props(b).Perimeter(1));
+        max_in = max(max_in, props(b).MaxIntensity(1));
```

Comment:
Sorry, to be a pain, could we have the image name as well please ? Note that the IDs are not ordered in the CSV - also, in the corresponding Fiji CSV, the names of images are present.